src/components/Profile/ProfileStyledLink.tsx

```diff
@@ -21,8 +12,8 @@ interface Props {
 const ProfileStyledLink = ({ children, className, href }: Props) => (
   <StyledLink
     className={className}
-    linkColor={linkColor}
-    hoverColor={hoverColor}
+    linkColor={colors.defaultHover}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)